Business/Concrete/ProductManager.cs

```diff
@@ -20,19 +21,24 @@ public ProductManager(IProductRepository productRepository, IMapper mapper)
         _mapper = mapper;
     }
 
+    [CacheRemoveAspect("IProductService.Get")]
     public async Task AddAsync(Product product) => 
         await _productRepository.AddAsync(product);
 
+    [CacheRemoveAspect("IProductService.Get", Priority = 1)]
     public async Task DeleteAsync(Product product) => 
         await _productRepository.DeleteAsync(p => p.Id == product.Id);
 
+    [CacheRemoveAspect("IProductService.Get")]
     public async Task UpdateAsync(Product product) => 
         await _productRepository.UpdateAsync(product);
 
+    [CacheAspect(duration: 30)]
```

Comment:
I strongly advise you to put it into constants since `30` is not too obvious: 30 seconds/milliseconds/minutes etc